### ISSUE TITLE
Mac py3.6 build is failing

### ISSUE THREAD

**Issue by jorisvandenbossche** [conda-forge/pandas-feedstock#16]
https://travis-ci.org/conda-forge/pandas-feedstock/builds/186561914

End of log says (https://travis-ci.org/conda-forge/pandas-feedstock/jobs/186561921):

```
creating build/temp.macosx-10.7-x86_64-3.6/pandas

gcc -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -arch x86_64 -Ipandas/src/klib -Ipandas/src -I/Users/travis/miniconda3/envs/_build_placehold_placehold_placehold_placehold_pla/lib/python3.6/site-packages/numpy/core/include -I/Users/travis/miniconda3/envs/_build_placehold_placehold_placehold_placehold_pla/include/python3.6m -c pandas/lib.c -o build/temp.macosx-10.7-x86_64-3.6/pandas/lib.o -Wno-unused-function

error: $MACOSX_DEPLOYMENT_TARGET mismatch: now "10.6" but "10.7" during configure

Command failed: /bin/bash -x -e /Users/travis/miniconda3/conda-bld/work/pandas-0.19.2/conda_build.sh
```

**Comment by jorisvandenbossche** [comment]
Apparently closed by #17

**Comment by ocefpaf** [comment]
Lately everything that fails I just switch it to `conda-build 2` before trying to debug it  :smile:
